app/src/main/res/layout/item_character.xml

```diff
@@ -2,24 +2,45 @@
 <androidx.constraintlayout.widget.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <ImageView android:id="@+id/image_character"
-        android:layout_width="80dp"
-        android:layout_height="80dp"
-        android:layout_margin="10dp"
+    <ImageView
+        android:id="@+id/image_character"
+        android:layout_width="150dp"
+        android:layout_height="150dp"
+        android:layout_margin="8dp"
         android:background="@drawable/huckpng"
         android:padding="40dp"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"/>
-        
-    
-    <TextView android:id="@+id/character_name"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/character_name"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        app:layout_constraintStart_toEndOf="@+id/image_character"
+        android:layout_marginLeft="8dp"
         android:text="Just a Test"
-        android:textSize="24sp"/>
+        android:textSize="34sp"
+        app:layout_constraintStart_toEndOf="@+id/image_character"
+        app:layout_constraintTop_toTopOf="@+id/image_character" />
+
+    <TextView android:id="@+id/character_description"
+        android:layout_width="239dp"
+        android:layout_height="94dp"
+        android:layout_margin="8dp"
+        android:text="another text to test it"
```

Comment:
Para vc fazer mock de texto, imagem etc... no layout, use o tools: ao invés de android: O android:text deve ser utilizado para o texto real. Então nesse caso do texto vc coloca tools:text="another text to test it"